blueprints/banking_blueprint/__init__.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+"""This module contains a MindMeld application"""
+from mindmeld import Application
+from mindmeld.models import entity_features
+from mindmeld.markup import process_markup
+from mindmeld.core import FormEntity
+
+app = Application(__name__)
+
+__all__ = ['app']
+
+
+sample_user = {
+    'savings' : 500,
+    'checking' : 100,
+    'credit' : 60,
+    'autoPay' : False,
+}
```

Comment:
Would be cool to add account numbers and name, username, and password here as well to make it look a bit more realistic.